pipelines/apDetectorVisitQualityCore.yaml

```diff
@@ -12,3 +12,10 @@ tasks:
       atools.simpleSky: SimpleDiaPlot
       python: |
         from lsst.analysis.tools.atools import *
+  analyzeTrailedDiaSrcCore:
+    class: lsst.analysis.tools.tasks.TrailedDiaSrcDetectorVisitAnalysisTask
+    config:
+      atools.numDiaSourcesAll: NumDiaSourcesAllMetric
```

Comment:
I _think_ this is counting "all" the DiaSources that are in the data product you configure in `TrailedDiaSrcDetectorVisitAnalysisConnections`. (AmazingCamelCaseNamingSituationBtw.) Is that right? It's confusing because it's named in such a way that it looks like you're counting the number of regular DiaSources. Not sure how to make it clearer.

Reply:
Not sure how to make this clearer either other than renaming the analysis tool. All this tool is count up the number of diaSources in a data product like you said. Maybe just a comment will do for clarity?